trustymail/trustymail.py

```diff
@@ -221,20 +215,16 @@ def spf_scan(domain):
                 logging.debug("\tResult Differs: Expected [{0}] - Actual [{1}]".format(result, response[0]))
                 domain.errors.append("Result Differs: Expected [{0}] - Actual [{1}]".format(result, response[0]))
```

Comment:
1. Should this be `domain.syntax_errors`?  
2. The field `domain.errors` doesn't even appear in the output of `trustymail`.  Should it?  Note that the method `handle_errors` below also writes to `domain.errors`.

Reply:
This a moot point now, since I added {{domain.errors}} to the outout of {{trustymail}}.